recipe/meta.yaml

```diff
@@ -19,15 +19,19 @@ source:
     - patches/0004-Prefer-getenv-TOPSRCDIR-over-STRINGIFY-TOPSRCDIR.patch
     - patches/0009-topsrcdir.patch
     - patches/prevent_MS_runtime_libs_being_installed_again.patch  # [win]
-    - patches/do_not_use_16_processes_in_tests.patch  # [ppc64le]
+    - patches/do_not_use_more_than_2_processes_in_tests.patch  # [linux]
     - patches/0001-Fix-issue-2674.patch
     - patches/0001-Switch-custom-Bzip2-cmake-module-to-standard.patch
     # removeme once 4.9.3 is released
     - patches/update_ftp_refs.patch
 
+
 build:
   number: {{ build }}
   skip: true  # [win and vc<14]
```

Comment:
Let's remove this for cleanliness while we're adding a new skip.  This version isn't being built anyway.